pavelib/utils/test/suites/acceptance_suite.py

```diff
@@ -38,10 +38,11 @@ def cmd(self):
 
         cmd = (
             "DEFAULT_STORE={default_store} ./manage.py {system} --settings acceptance harvest --traceback "
-            "--debug-mode --verbosity {verbosity} {report_args} {extra_args}".format(
+            "--debug-mode --verbosity {verbosity} {pdb} {report_args} {extra_args}".format(
```

Comment:
It's not functionally a problem, but it would be nice to not have extra spaces randomly in the logs.